src/brain-gain/main.css

```diff
@@ -31,10 +39,48 @@ core-navbar a:hover,
 core-navbar a:active {
   transform: scale(1.4);
 }
+core-navbar b:hover,
+core-navbar b:active {
+  transform: scale(1.2);
+}
 core-navbar > div {
   padding: 8px;
 }
 
+/* best scores for all games */
```

Comment:
For now the best game bannar could be blocked by the other game buttons. It will be nice to use "z-index" propertity on the modal that it can overlay other bottons.
![image](https://user-images.githubusercontent.com/79813230/192943451-cca5daad-5ece-46fc-9434-7d4b031b4a53.png)

Reply:
resolved